src/command_modules/azure-cli-resource/setup.py

```diff
@@ -27,6 +27,7 @@
 DEPENDENCIES = [
     'azure-mgmt-resource==1.0.0rc1',
     'azure-cli-core',
+    'azure-mgmt-authorization'
```

Comment:
The command doesn't have specific version requirements, so I leave the version range open, but let me know if there are any concerns.